### ISSUE TITLE
Not yet published?

### ISSUE THREAD

**Issue by augnustin** [swoosh/phoenix_swoosh#140]
So I tried to upgrade to `swoosh 1.0` (congrats! :tada: )

But when I did that, I got:

```
mix deps.get
Resolving Hex dependencies...

Failed to use "swoosh" (version 1.0.0) because
  phoenix_swoosh (version 0.2.0) requires ~> 0.1
  mix.exs specifies ~> 1.0.0

** (Mix) Hex dependency resolution failed, change the version requirements of your dependencies or unlock them (by using mix deps.update or mix deps.unlock). If you are unable to resolve the conflicts you can try overriding with {:dependency, "~> 1.0", override: true}
```

But if I look at `phoenix_swoosh`'s [`mix.exs`](https://github.com/swoosh/phoenix_swoosh/blob/master/mix.exs), there is:

```
  defp deps do
    [{:swoosh, "~> 1.0"},
    ...
  end
```

Any idea why? Maybe `phoenix_swoosh` upgrade hasn't been published yet?

Cheers

**Comment by princemaple** [comment]
@stevedomin could you give me access to publish swoosh_phoenix?

**Comment by princemaple** [comment]
Published v0.3

Please open new issues if there is anything not working :) Closing this one for now
